src/components/Select/__snapshots__/test.js.snap

```diff
@@ -193,9 +193,6 @@ exports[`Select matches snapshot: enzyme.mount 1`] = `
   flex-direction: column;
   box-sizing: border-box;
   width: 100%;
-  border-left: ;
```

Comment:
I wonder if there's a way we can have a unit test that validates that css output validates

Reply:
would be good to get css linting working, but the plugins for `css-in-js`/glamorous aren't mature enough/working as expected